fvm/environment/meter.go

```diff
@@ -54,7 +54,7 @@ const (
 	ComputationKindEncodeEvent                = 2040
 	_                                         = 2041
 	ComputationKindEVMEncodeABI               = 2042
-	ComputationKindEVMDecodeABI               = 2042
```

Comment:
@onflow/flow-execution-and-verification maybe we can avoid such mistakes by refactoring this to "base + iota", e.g.

```go 
const (
	// [2_000, 3_000) reserved for the FVM
	ComputationKindHash = 2_001 + iota
	ComputationKindVerifySignature
	ComputationKindAddAccountKey
	// ...
```

Reply:
Updated in https://github.com/onflow/flow-go/pull/5146/commits/12d7b37ff303601c9be89d3427e6fa62bb2c12d3
If it's out of the scope of the PR, I can exclude the commit though.

Reply:
I think "leaving it better than you found it" when touching code is a good practice, so thank you